docs/api/1.x/index.rst

```diff
@@ -12,18 +12,22 @@ Full detailed API documentation:
    :maxdepth: 2
 
    authentication
+   resource
```

Comment:
> My hope is that we will feel embarrassed by their exposure and that will make us fix them :) 

That can be a strategy indeed! ... audacious but it may work :)